tests/installation/ipxe_install.pm

```diff
@@ -195,8 +195,10 @@ sub set_bootscript_agama_cmdline_extra {
         $cmdline_extra .= "agama.install_url=$agama_install_url ";
     }
     if (is_ipmi) {
-        my $sol_console = get_required_var('IPXE_CONSOLE');
-        $cmdline_extra .= "console=$sol_console linuxrc.log=/dev/$sol_console linuxrc.core=/dev/$sol_console linuxrc.debug=4,trace ";
+        my $ipxe_console = get_required_var('IPXE_CONSOLE');
+        my @elements = split(/,/, $ipxe_console);
+        my $sol_console = $elements[0];
```

Comment:
`my $sol_console = (split(/,/, $ipxe_console))[0];` will do the job without elements. or even `my $sol_console = (split(/,/, get_required_var('IPXE_CONSOLE')))[0]`

Reply:
Apart from this remark, it looks good to me